src/cloudant/account.py

```diff
@@ -404,25 +399,24 @@ def _usage_endpoint(self, endpoint, year=None, month=None):
 
     def bill(self, year=None, month=None):
         """
-        _bill_
-
-        Get your cloudant billing data, optionally for a given year/month
+        Retrieves Cloudant billing data, optionally for a given year/month.
 
-        :param year: int, year, eg 2014
-        :param month: int, 1-12
+        :param int year: Year to query against, eg 2014.  Defaults to None
```

Comment:
Changed to "for example" throughout.  It actually matches our coding sample as well.  We use "for example" there.  Done in b19046696b22d6b3fb89e04d7d4b7a86ca7d9c41.